app/src/main/java/org/schabi/newpipe/fragments/list/playlist/PlaylistFragment.java

```diff
@@ -328,9 +332,13 @@ public void handleResult(@NonNull final PlaylistInfo result) {
                 && (YoutubeParsingHelper.isYoutubeMixId(result.getId())
                 || YoutubeParsingHelper.isYoutubeMusicMixId(result.getId()))) {
             // this is an auto-generated playlist (e.g. Youtube mix), so a radio is shown
-            headerBinding.uploaderAvatarView.setDisableCircularTransformation(true);
-            headerBinding.uploaderAvatarView.setBorderColor(
-                    getResources().getColor(R.color.transparent_background_color));
+            final ShapeAppearanceModel model = ShapeAppearanceModel.builder()
+                    .setAllCorners(CornerFamily.ROUNDED, 0f)
+                    .build(); // this turns the image back into a square
+            headerBinding.uploaderAvatarView.setShapeAppearanceModel(model);
+            headerBinding.uploaderAvatarView.setStrokeColor(ColorStateList.valueOf(
+                    getResources().getColor(R.color.transparent_background_color))
```

Comment:
Not sure why this is formatted like this.
I think something like this would be better:
```suggestion
            headerBinding.uploaderAvatarView.setStrokeColor(
                   ColorStateList.valueOf(
                          getResources().getColor(R.color.transparent_background_color))
```

Reply:
Sure 👍